### ISSUE TITLE
Windows gem install "Can't Find ImageMagick Library" Error Output misformatted

### ISSUE THREAD

**Issue by slime-girl** [rmagick/rmagick#1295]
### Description
When the gem installer fails at the step: `searching PATH for the ImageMagick library...`
it will print out the following error message:

```
ERROR: Can't install RMagick 4.2.2.
Can't find the ImageMagick library.
Retry with '--with-opt-dir' option.
Usage: gem install rmagick -- '--with-opt-dir="[path to ImageMagick]"'
e.g.
  gem install rmagick -- '--with-opt-dir="C:Program FilesImageMagick-6.9.1-Q16"'
```

This issue is for the very last line of that output: the path separators are missing, i presume escaped out because they would be backslashes on windows.

### Steps to Reproduce
`gem install rmagick` while:
1. having `magick` installed and in the `PATH`
2. NOT having the ImageMagick Library available in the `PATH`

### System Configuration
I am running windows 10.
I have installed `magick`.

- ImageMagick version: `7.1.0`
- RMagick version: `latest`
- Ruby version: `3.0.2`
- Environment (Operating system, version and so on): Windows 10
- Additional information:
  I don't know if this repo does a `Good First Contribution` label but if it does I suggest this issue ;)

**Comment by mockdeep** [comment]
Hi @slime-girl, thanks for reporting. I'm not sure we'll be able to get to this any time soon. Do you by any chance want to have a stab at fixing it?

**Comment by Watson1978** [comment]
> NOT having the ImageMagick Library available in the PATH

You should set  PATH environment variable properly with directory where ImageMagick is installed.

**Comment by slime-girl** [comment]
for clarity, (and for someone finding this issue via google in 4 years or so) , my install problem was most likely because i had installed ImageMagick without checking the "with C and C++ dev headers" box. I think. The install folder was in `$PATH` it just seems the gem installer couldn't find something it needed in that folder. When I reinstalled and made sure i checked that C and C++ toggle, `gem install` worked without issue. 

That said, this issue isn't about debugging my installation. It's about fixing an error output for clarity. I have approved the PR by @dlemstra . Thanks!
